content/en/network_monitoring/devices/netflow.md

```diff
@@ -0,0 +1,56 @@
+---
+title: NetFlow Monitoring
+kind: documentation
+further_reading:
+- link: "/network_monitoring/devices/profiles"
+  tag: "Documentation"
+  text: "Using Profiles with Network Device Monitoring"
+---
+
+{{< beta-callout url="#" btn-hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.
+{{< /beta-callout >}}
+
+## Overview
+
+Use NetFlow Monitoring in Datadog to visualize and monitor your flow records from your Netflow-enabled devices.
+
+## Installation
+
+To use NetFlow Monitoring with Network Device Monitoring, ensure you are using the [Agent][1] version 7.40 or newer.
+
+**Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is recommended.
+
+## Configuration
+
+To configure your devices to send NetFlow, sFlow, or IPFIX traffic to Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```

Comment:
```suggestion
To configure your devices to send NetFlow, sFlow, or IPFIX traffic to Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```
Should this be "the/an" Agent NetFlow Server? Or is there only one thing called Agent NetFlow Server where all the traffic goes?